docs/profiles.md

```diff
@@ -130,7 +130,7 @@ Attribute       | Type    | Notes
 barrier         | Boolean | Is it an impassable barrier?
 traffic_lights  | Boolean | Is it a traffic light (incurs delay in `process_turn`)?
 
-## process_way(profile, way, result)
+### process_way(profile, way, result)
```

Comment:
form the code `process_way` has the 4th `relations` argument but here it is not mentioned

Should it be may be optional ? I feel we need some sort of sync here with location-dependent tags, that I would like to see optional. So we can add an additional positional argument that will hold a table with optional data items.
This will save from bumping api version every time when a new piece of data will be added to lua callback functions.